File: dojo/th.m/FizzBuzz/Ruby/01-12-2014 Green/fizzbuzz.rb
def fizzbuzz(from,to)
  for x in from..to
    output = ""
    output += "Fizz" if x%3 == 0
    output += "Buzz" if x%5 == 0
    output = x.to_s if output == ""
    puts output
  end
end

fizzbuzz(1,100)

File: dojo/th.m/FizzBuzz/PHP/01-12-2014 Extra/fizzbuzz.php
<?php
/**
* FizzBuzz Kata
* Thorne Melcher
* 01/12/2014
*
* I haven't done this in PHP yet, so why not? And I'll have fun with tertiary
* logic to keep the line count down!
*/
for($i=1; $i<=100; $i++)
{
  $output = ($i%3==0 ? "Fizz" : "") . ($i%5==0 ? "Buzz" : "");
  if($output == "") $output = $i;
  echo $output . "<br />";
}